services/second-service/src/posts/dto/posts.dto.ts

```diff
@@ -17,3 +18,25 @@ export class PostsDto {
   )
   createdAt: Date | null;
 }
+
+export class PaginationDto {
+  total: number;
+  limit: number;
+  offset: number;
+}
+
+export class PostsWithPaginationDto {
+  @Expose()
+  posts: PostsDto[];
+
+  @Expose()
+  pagination: PaginationDto;
+}
+
+export class GetPostsDto {
+  @IsNotEmpty()
+  limit: number;
```

Comment:
I would suggest adding defaults here

Reply:
But I could be wrong

Reply:
I can add defaults, but anyway I have to pass the param value because it's required.